src/js/_book.html

```diff
@@ -19,12 +19,18 @@ <h3 class="author"><%= data.book.author %></h3>
   <%= data.book.text %>
   <div class="reviewer">
     &mdash;
-    <% if (data.reviewer.link) { %>
-    <a href="<%= data.reviewer.link %>"><%= data.book.reviewer %></a>,
+    <% if (data.reviewer) { %>
+    <% (data.reviewer).forEach(function(rev, i) { %>
+    <% if (rev.link) { %>
+    <a href="<%= rev.link %>"><%= rev.key %></a>,
     <% } else { %>
-    <%= data.book.reviewer %>,
+    <%= rev.key %>,
+    <% } %>
+    <%= poorMD(rev.title || "NPR Staff") %>
+    <% if(i == 0) {%> and
```

Comment:
This works well if we have two reviewers. It's going to be somewhat less workable if we have three. Let's code from the start under the assumption that supporting multiple reviewers means supporting AP-style comma lists.